### PR TITLE
Remove superfluous check

### PR DESCRIPTION
**Pull request by rowleya**
No need to check our own maths unless we think we are wrong... and our check was wrong...

### CODE REVIEW

File: spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py [hunk 1]
```diff
@@ -425,12 +419,10 @@ def _generate_prefix(virtual_key, prefix_type):
 
     @staticmethod
     def _calculate_mask(n_neurons):
-        if n_neurons == 1:
-            return 0xFFFFFFFF, 1
         temp_value = int(math.ceil(math.log(n_neurons, 2)))
         max_key = (int(math.pow(2, temp_value)) - 1)
```
Comment (dkfellows) [inline spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py]: `max_key = 2**temp_value - 1`

**Comment by dkfellows** [comment]
Some experimentation indicates that doing it the way I suggest computes identical values, but in about a third of the time. I think I'm OK with that.

**Comment by rowleya** [comment]
What? More efficient code? Why would we do that? Oh, OK, I suppose so... ;).

File: spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py [hunk 2]
```diff
@@ -425,12 +419,10 @@ def _generate_prefix(virtual_key, prefix_type):
 
     @staticmethod
     def _calculate_mask(n_neurons):
-        if n_neurons == 1:
-            return 0xFFFFFFFF, 1
         temp_value = int(math.ceil(math.log(n_neurons, 2)))
```
[the remark this answers is not in the document]
Comment (rowleya) [reply spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py]: Ooh didn't know about that one...

File: spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py (hunk 1 above)
Comment (rowleya) [reply spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py]: Yep, makes sense!

File: spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dkfellows) [reply spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py]: Amazing what you find when you go rummaging through the API docs and searching on SO…

**Pull request merged**